dependencies/pip/external_services.txt

```diff
@@ -85,7 +84,7 @@ pytest==3.0.3             # via pytest-django
 python-dateutil==2.6.0
 python-digest==1.7
 pytz==2016.4
-pyxform==0.11.5
+pyxform==0.12.0
```

Comment:
`make pip_compile` made this change. Seemed fine to me since `requirements.txt` and `dev_requirements.txt` already had it.